### ISSUE TITLE
Community Funding Proposals

### ISSUE THREAD

**Issue by realcodywburns** [ethereumproject/volunteer#11]
Before I begin this issue I fully acknowledge both the irony and optics of my suggestion.

At some point in the near future, projects will require funding to continue. The current method of community funding has seemingly worked well for the webpage. However, less public projects will require more TLC than public facing sides. As such, I believe we need to form a 'blockchain based congress' in order to fund ethereum<sup>classic</sup> specific projects. Having a smart contract holding community funds would allow investors to fund the community knowing that at least some sort of vote was given on the usage of the capital. 

As outlined in the smart contract documentation (https://www.ethereum.org/dao) , a voting congress could be formed by a president and community members added to vote on issues. A git repository could be added to ethereumproject or ethereumclassic  where proposals and budgets could be submitted and then, at an agreed upon time, the congress would hold a vote and direct funds. The president would specifically be barred from voting and serve in a 'collect & collaborate' and rather than a 'command & control' role. 

The failing of the pre-fork DAO was due to poor and overreaching coding. The Blockchain congress should have two documents: 1) the code 2) a document specifically outlining the intent of the code in plain English. These documents will severe as the Articles of Confederation (https://en.wikipedia.org/wiki/Articles_of_Confederation) for the community.

**Comment by whatisgravity** [comment]
I have posted about my interest in using a smart contract to transparently manage donated money to the development effort.

I'm not entirely convinced on the idea of a general investment fund but I think there may be merit to the idea that I'm unable to understand. I'm mostly wary of unnecessary crowd funding and all the cascading affects funding methods has on a project. 

The most important thing in my opinion is that for either design, we need to start simple and add complexity in later iterations. Boil the concept down to the base components so we can build the simplest iteration because having some basic structure now is more valuable than having a more complex structure that would be cumbersome to launch.

**Comment by realcodywburns** [comment]
Yes, a contract which is very limited in scope and with a publicly know purpose is what i am proposing. the code available for review before being deployed. It also need to be known that this is not intended for governance in any form other, just for funding development.

**Comment by whatisgravity** [comment]
I think this is a good idea and I would like to donate to the development of ETC once a basic structure exists. If nothing is either outlined or mocked up by the time I finish Mist Wallet, I can put together an early mock up of my initial idea regarding this. 

I'm not particularly attached to this idea because I think there are very likely better designs but my initial design has 2 user maps. 

1. The first user map would be the key holders, x number of the signatures would be required for a payment to be processed. More could be required to dissolve the organization.

2. The second user map would be all active developers or other members who are white listed to received payment from the trust. The white list would be controlled by the first user map. 

I'm not entirely sure how to fairly determine the first user map of key holders.  The existing key holders and possibly white listed developers should be able to vote in new members, and members can choose to resign. 

The system needs to be analyzed critically so we don't leave ways to easily game the system. We should also likely institute a limit early on to prevent too much from being held in the trust. 

This method requires trust of the key holders, but it divides the trust across the people taking leadership roles in the community development initiative. It is also completely transparent which hopefully would make it easier to spot any corruption or attempts to game the system.

I'm interested in other ideas if anyone wants to share a description of their design or a mock up of their proposed design.

**Comment by realcodywburns** [comment]
I have attached a framework here

https://github.com/ethereumproject/Project_Funding

The smart contract is just a copy/paste

**Comment by whatisgravity** [comment]
Conversation has migrated to: https://github.com/ethereumproject/project-funding/issues/7

I encourage everyone to participate, this is a very important topic.
